builds/mariner/SPECS/aziot-edge/aziot-edge.spec

```diff
@@ -0,0 +1,189 @@
+%define iotedge_user iotedge
+%define iotedge_group %{iotedge_user}
+%define iotedge_home %{_localstatedir}/lib/aziot/edged
+%define iotedge_logdir %{_localstatedir}/log/aziot/edged
+%define iotedge_socketdir %{_localstatedir}/lib/iotedge
+%define aziot_confdir %{_sysconfdir}/aziot
+%define iotedge_confdir %{aziot_confdir}/edged
+
+Name:           aziot-edge
+Version:        @@VERSION@@
+Release:        3%{?dist}
+
+License:        MIT
+Summary:        Azure IoT Edge Module Runtime
+URL:            https://github.com/azure/iotedge
+
+%{?systemd_requires}
+BuildRequires:  systemd
+BuildRequires:  rust >= 1.47.0
+Requires(pre):  shadow-utils
```

Comment:
We requires the IIS component here (with a strict version)
`Requires:       aziot-identity-service = 1.2.2-1`

Reply:
Changed to now require aziot-identity-service